src/Symfony/Component/Debug/ErrorHandler.php

```diff
@@ -282,7 +282,7 @@ public function scopeAt($levels, $replace = false)
     }
 
     /**
-     * Sets the error levels that are logged with their stack trace.
+     * Sets the PHP error levels for which stack trace are preserved.
```

Comment:
Typo. Replace it by either `... stack trace is preserved.` or `... stack traces are preserved.`